### PR TITLE
add docker-compose.yml

### PR DESCRIPTION
**Pull request by mchangrh**
with ENV categories, it can be launched with docker-compose

### CODE REVIEW

**Comment by mchangrh** [comment]
Tested & working on windows & armv7

**Comment by erdnaxeli** [comment]
Could you move it in the `contrib/` folder please?

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: '3.3'
+services:
+  castblock:
+    network_mode: host
+    image: erdnaxeli/castblock
+    environment:
+      - CATEGORIES=sponsor,selfpromo
+      # Full list of categories https://github.com/ajayyy/SponsorBlock/wiki/Types#category
```
Comment (erdnaxeli) [inline docker-compose.yml:8]: a return is missing at the end of this line

Comment (mchangrh) [reply docker-compose.yml]: Fixed

**Review by erdnaxeli — APPROVED** [review]

**Pull request merged**